src/pages/index.js

```diff
@@ -21,73 +21,229 @@ import {
   profilePicture,
   openModal,
   closePopup,
+  profileAvatarEditButton,
   options,
 } from "../utils/constants.js";
+import Api from "../components/Api.js";
+import PopupWithConfirmation from "../components/PopUpWithConfirmation.js";
 
 // Data
 const initialCards = [
   {
     name: "Yosemite Valley",
     description: "Yosemite Valley",
     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/yosemite.jpg",
+    _id: "card1",
   },
   {
     name: "Lake Louise",
     description: "Lake Louise",
     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lake-louise.jpg",
+    _id: "card12",
   },
   {
     name: "Bald Mountains",
     description: "Bald Mountains",
     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/bald-mountains.jpg",
+    _id: "card123",
   },
   {
     name: "Latemar",
     description: "Latemar",
     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/latemar.jpg",
+    _id: "card4",
   },
   {
     name: "Vanoise National Park",
     description: "Vanoise National Park",
     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/vanoise.jpg",
+    _id: "card5",
   },
   {
     name: "Lago di Braies",
     description: "Lago di Braies",
     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lago.jpg",
+    _id: "card6",
   },
 ];
 //
+// API initialization
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "f10ef152-cb9d-40bf-b92c-7c27e5b2bf36",
+    "Content-Type": "application/json",
+  },
+});
+
+// User Info
+const userInfo = new UserInfo({
+  profileTitle: document.querySelector(".profile__title"),
+  profileDescription: document.querySelector(".profile__description"),
+  profilePicture: document.querySelector(".profile__image"),
+});
+
+// Form Validators
+const profileEditFormValidator = new FormValidator(options, profileEditForm);
+profileEditFormValidator.enableValidation();
+
+const addCardFormValidator = new FormValidator(options, addCardFormElement);
+addCardFormValidator.enableValidation();
+
+const avatarEditFormValidator = new FormValidator(
+  options,
+  document.querySelector("#change__profile_picture")
```

Comment:
`  document.querySelector("#change__profile_picture")
`

**Could be improved**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms